src/layout/msWorkspace/msWorkspace.js

```diff
@@ -244,15 +246,36 @@ var MsWorkspace = class MsWorkspace {
         this.focusTileable(this.tileableList[this.focusedIndex - 1]);
     }
 
-    focusAppLauncher() {
-        this.focusTileable(this.tileableList[this.tileableList.length - 1]);
+    focusAppLauncher(usedHotkey) {
+        if (
+            !this.tileableList ||
+            this.tileableList.length < 2 ||
+            this.focusTileable === this.appLauncher
+        )
+            return;
+        this.focusTileable(this.appLauncher);
+        this.lastFocusUsedHotkey = usedHotkey;
```

Comment:
I don't think it's should be related to if we used the hotkey to reach it but instead if we called this function. Currently the only call to focusAppLauncher is from the hotkey anyway